src/classes/game.rb

```diff
@@ -1 +1,17 @@
-# Game class
+# Game class
+require_relative './item'
+require 'date'
+class Game < Item
+  attr_reader :multiplayer, :last_played_at
+
+  def initialize(publish_date, multiplayer, last_played_at)
+    super(publish_date)
+    @id = rand(1..1000)

```

Comment:
Kindly don't override `@id` value initialization because you have already done this process in the Item class.